src/sparseml/benchmark/info.py

```diff
@@ -544,6 +545,11 @@ def _parse_args():
 
 
 def _main():
+    warnings.warn(
```

Comment:
use to handle setting the warning (by default deprecation warnings are suppressed): https://github.com/neuralmagic/sparseml/blob/c3612a19b16a3d60493ff5447c5c860c16e172df/src/sparseml/utils/helpers.py#L827